plugins/scaffolder/alpha-api-report.md

```diff
@@ -60,11 +74,64 @@ export const nextSelectedTemplateRouteRef: SubRouteRef<
   PathParams<'/templates/:namespace/:templateName'>
 >;
 
+// @public (undocumented)
+export const registerComponentRouteRef: ExternalRouteRef<undefined, true>;
+
+// @alpha
+export const RegisterExistingButton: (
```

Comment:
I'm also in two minds to export this too - feels pretty small and maybe not worth it?